lambda/ecr_scan_notifications.tf

```diff
@@ -11,6 +11,7 @@ resource "aws_lambda_function" "ecr_scan_notifications_lambda_function" {
   environment {
```

Comment:
Has `terraform fmt` been run on this file?

Reply:
Sorry, no. I've run it now.